vars/base/kubernetes_1_31.json

```diff
@@ -1,6 +1,6 @@
 {
-    "kubernetes_deb_version": "1.31.8-*",
-    "kubernetes_rpm_version": "1.31.8",
-    "kubernetes_semver": "v1.31.8",
-    "kubernetes_series": "v1.31"
-}
\ No newline at end of file
+  "kubernetes_deb_version": "1.31.5-*",
+  "kubernetes_rpm_version": "1.31.5",
+  "kubernetes_semver": "v1.31.5",
+  "kubernetes_series": "v1.31"
+}
```

Comment:
NIT: I think this is auto generated, so I am not sure if the scripts will generate bad values here?

Reply:
Good spot - have patched the scripts that generate these to make linter-compliant output